cypress/e2e/spec.cy.ts

```diff
@@ -0,0 +1,58 @@
+describe('Ethereum', () => {
+  before(() => {
+    cy.visit('/')
+    cy.url().should('include', 'swap/ethereum')
+    cy.get('#walktour-tooltip-container svg').click()
+  })
+
+  describe('Swap Page', () => {
+    it('Should be search token by symbol successfully', () => {
+      cy.get('#swap-currency-input .token-symbol-container').click()
+      cy.get('#token-search-input').type('KNC')
+      cy.get("[title='Kyber Network Crystal v2']").first().click({ force: true })
+      cy.url().should('include', 'swap/ethereum/knc-to-usdt')
+    })
+  })
+
+  describe('Pools page', () => {
+    it('should show list pools', () => {
+      cy.visit('/pools/ethereum')
+      cy.get('.iskUse div a').should('have.length.at.least', 24)
```

Comment:
`iskUse` looks like a generated class. If so, please avoid this as it will be a different one when a new version of KyberSwap is released